src/syscheckd/db/src/file.cpp

```diff
@@ -83,7 +83,7 @@ FIMDBErrorCode fim_db_get_path(const char* file_path, callback_context_t callbac
             }
             else
             {
-                const auto file { std::make_unique<FileItem>(entry_from_path) };
+                const auto file { std::make_unique<FileItem>(entry_from_path[0]) };
```

Comment:
I understand that getDBItem returns an array now, if the expected behavior is an array of only one element, here we should control it by checking the size, We could throw a runtime_error exception in case there are more..

Apart from this, it is preferable to use entry_from_path.front() (if quotes are added to it, use entry_from_path.front().get_ref<const std::string &>()) instead of entry_from_path [0]

Reply:
Done